### DISCUSSION TITLE
The question of 7B MiniGPT-4 checkpoint with only stage one training

### DISCUSSION THREAD

**Discussion by slacklife** [Vision-CAIR/MiniGPT-4#367] [General]
Great work!

I want to figure if the  [7B MiniGPT-4 checkpoint ](https://github.com/Vision-CAIR/MiniGPT-4#training)with only stage one training is trained with vicuna7B or llama-2-7b-chat.
I want the checkpoint trained with llama-2-7b-chat and only trained with only stage one.

Thanks!

**Comment by wyuzh** [comment]
Same question, I use the provided 7B MiniGPT-4 checkpoint for the second FT with Llama2, but it gives error `size mismatch for llama_proj.weight: copying a param with shape torch.Size([4096, 768]) from checkpoint, the shape in current model is torch.Size([4096, 5632]).`  However, it works fine when I use the ckpt for second FT with Vicuna 7B, may be this ckpt is for Vicuna 7B only. Could you please provide the first stage ckpt that works for Llama2. Thank you very much!
